markdown-templates/intern-intro-email-template.md

```diff
@@ -0,0 +1,102 @@
+# Email template: Intern Introduction
+
+Dear {APPLICANT NAME},
+
+Welcome to the JupyterHub community! We are so excited to have you here and begin
+working with you on your Outreachy project: **{PROJECT TITLE}**. This email will
+put you in touch with your primary mentor {MENTOR NAME}, if you are not already,
+and provide some instructions for getting started in your first week.
+
+## Meeting Notes
+
+We have provided a document for you to keep notes from your mentor meetings in.
+It's very important that you keep notes so that you can remember what was
+discussed, which may help in planning next issues for you to work on or writing
+up blog posts.
+
+- Notes link: {NOTES LINK}
+
+This layout is for **guidance only**. If you prefer a different format, you are
+free to change the document to suit you.
+
+In the JupyterHub community, we use HackMD (https://hackmd.io) for collaborative
+notetaking, which is the platform your notes document has been created on. If
+you are new to HackMD, you can read this short guide from Open Life Science on
+how to use it: https://hackmd.io/@openlifescience/OLS-HackMD-guide However, this
+is also for guidance only, and if using another platform, like Google Docs,
+works better - that's ok too!
+
+## Setup your Mentor Meetings
+
+Please reach out to your mentor (cc'd on this email) and arrange a suitable
+recurring day, time, and platform for your mentor meetings.
+
+- **Day and Time:** Remember that you and your mentor may not be in the same
+  timezone, so try to be considerate of early mornings or late evenings. We also
+  highly encourage taking the weekends off to rest and recharge!
+- **Frequency:** Outreachy recommends meeting with your mentor weekly for about
+  30 minutes, so make sure the day and time you pick is one that is generally
+  available every week.
```

Comment:
I need to verify this claim, but the outreachy site is currently down with certificate issues